src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -175,6 +189,13 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedValues(['ignore', 'ensure_single_line', 'ensure_fully_multiline'])
                 ->setDefault('ensure_fully_multiline')
                 ->getOption(),
+            (new FixerOptionBuilder(
+                'on_nested_multiline',
```

Comment:
The option name can be confusing because it's the same pattern as `on_multiline`. One could think enabling only this option will have some effect on the behavior of the rule, while its purpose is to change how the option `on_multiline` works. What about `consider_nested_multiline`?